src/main/java/com/minecolonies/coremod/colony/workorders/AbstractWorkOrder.java

```diff
@@ -112,6 +113,7 @@ public static AbstractWorkOrder createFromNBT(@NotNull final NBTTagCompound comp
         }
         try
         {
+        	order.setPriority(compound.getInteger(TAG_TH_PRIORITY));
```

Comment:
if (compound.hasKey(TAG_TH_PRIORITY))
Also needed here

Reply:
If this is the save, why are you doing compound.getInteger()?